lighthouse-viewer/README.md

```diff
@@ -20,3 +20,18 @@ yarn deploy-viewer
 ```
 
 For more information on deployment, see `releasing.md`.
+
+## PSI
```

Comment:
can you add this as  a gist example URL http://localhost:8000/?gist=bd1779783a5bbcb348564a58f80f7099